docs/sphinx/specification/quake-dialect.md

```diff
@@ -28,7 +28,7 @@ greater, the extra qubits are considered controls.
 The main motivation behind Quake's value model is to directly expose
 quantum and classical data dependencies for optimization purposes,
 i.e., to represent the dataflow in quantum computations.  In contrast
-to Quake's memory model, which uses memory semantics (quantum
```

Comment:
Quake is flexible in that it can represent both the memory and value semantics model (switching via the --memtoreg pass). So the text as-is is correct. Granted, the whole document here could use a refresh probably.